manageiq-operator/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Build the manager binary
+FROM docker.io/library/golang:1.18 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY main.go main.go
+COPY api/ api/
+COPY controllers/ controllers/
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
```

Comment:
will this arch matter for multi-arch builds?

Reply:
Removed it, it doesn't seem to matter on my machine